online-fishing-store/src/app/pages/main-page/products/products.component.ts

```diff
@@ -8,8 +8,12 @@ import { Product } from 'src/app/interfaces/product';
 })
 export class ProductsComponent implements OnInit {
   @Input() public products: Product[];
-
+  @Output() public selectProductEvent: EventEmitter <number> = new EventEmitter<number>();
```

Comment:
You don't need to add  "Event" word for each event. It is clear that it is event